src/Stack.cpp

```diff
@@ -0,0 +1,153 @@
+#pragma once
+#include "Stack.h"
+
+
+Stack::Stack(const size_t& size, const int& value)
+	:Vector::Vector(size, value)
+{
+}
+
+// Copy constructor
+Stack::Stack(const Stack& other) {
+	*this = other;//use Vector Copy Assiment
+}
+
+// Move Constructor
+Stack::Stack(Stack&& other) noexcept
+{
+	Vector::operator=(other);
+}
+
+Stack::~Stack()
```

Comment:
This is not good, implementing an empty destructor means no resources will be freed - specifically the array you allocated